sympl/_components/plot.py

```diff
@@ -64,13 +63,13 @@ def store(self, state):
         """
         if self.interactive:
             self._fig.clear()
-            time.sleep(1e-5)
             fig = self._fig
         else:
             fig = plt.figure()
         self._plot_function(fig, copy_state(state))
-        plt.draw_all()
+        plt.tight_layout()
+        fig.canvas.draw()
         if self.interactive:
```

Comment:
We should refactor this to use `if not self.interactive`, to avoid the `pass` block.